### ISSUE TITLE
Library is required to be in the "master" branch of the github repo.

### ISSUE THREAD

**Issue by zfields** [spark/uber-library-example#14]
You require the library to be in the master branch of the github repo. This isn't very conducive to existing repositories. I actually had to mangle the master branch of my existing library.

Could you guys give the option or make another rule to pull from a "spark" branch? It would allow me to make a commit that flattens my folder hierarchy, and a second that corrects my header inclusions. Then I could simply rebase those changes onto my master branch and leave them in the spark branch.

**Comment by zfields** [comment]
This issue is valid as of 2015/02/16. I'm cleaning my inbox and translating all my notes into issues.

**Comment by embedded-creations** [comment]
I agree with this feature request, and add that the README should mention that the "master" branch is the only branch that can be used.  When trying to validate a library with Particle Build, it just displays a blank screen when there is no master branch.
